scripts/list_workspace_resource.sh

```diff
@@ -759,6 +759,25 @@ function _list_cloudwatch_dashboards() {
   done
 }
 
+function _delete_cloudwatch_dashboards() {
+  local workspace=$1
+  local dashboards=$(aws cloudwatch list-dashboards --output json)
+
+  if [ -n "$workspace" ]; then
+    dashboards=$(echo "$dashboards" | jq -r --arg SUBSTRING "$workspace" '.DashboardEntries[] | select(.DashboardName | contains($SUBSTRING)) | .DashboardName')
+  else
```

Comment:
Don't need this conditional. We will never want to use this to delete everything.